src/lineup/internal/scorecmds.ts

```diff
@@ -54,12 +55,12 @@ export async function removeScoreImpl(inputs: IObjectRef<IViewProvider>[], param
   columnIds.forEach((id) => view.removeTrackedScoreColumn(id));
 
   return {
-    inverse: addScore(inputs[0], 'Score', parameter.id, parameter.params)
+    inverse: addScore(inputs[0], i18n.t('tdp:core.lineup.scorecmds.score'), parameter.id, parameter.params)
   };
 }
 
 export function addScore(provider: IObjectRef<IViewProvider>, scoreName: string, scoreId: string, params: any) {
-  return action(meta(`Add ${scoreName}`, cat.data, op.create), CMD_ADD_SCORE, addScoreImpl, [provider], {
+  return action(meta(`${i18n.t('tdp:core.lineup.scorecmds.add')} ${scoreName}`, cat.data, op.create), CMD_ADD_SCORE, addScoreImpl, [provider], {
```

Comment:
```suggestion
  return action(meta(i18n.t('tdp:core.lineup.scorecmds.add', {scoreName}), cat.data, op.create), CMD_ADD_SCORE, addScoreImpl, [provider], {
```

I would add the `scoreName` to the language string. Same goes for `${i18n.t('tdp:core.lineup.scorecmds.remove', {scoreName})}` below.

Reply:
Thank you for the feedback.